server/src/main/java/forklift/ForkliftServer.java

```diff
@@ -327,6 +326,6 @@ private ForkliftConnectorI startAndConnectToBroker() throws Exception {
             broker.start();
         }
         log.info("Connected to broker on " + brokerUrl);
-        return new ActiveMQConnector(brokerUrl);
+        return new ActiveMQConnector(brokerUrl, opts.getUsername(), opts.getPassword());
```

Comment:
Does this still work when `username` and `password` are null?

Reply:
Yes, I tested without passing in username and password and it ran without errors.